src/bio/bio_monitor.c

```diff
@@ -237,19 +237,61 @@ get_spdk_identify_ctrlr_completion(struct spdk_bdev_io *bdev_io, bool success,
 	spdk_bdev_free_io(bdev_io);
 }
 
+static int
+populate_dev_health(struct bio_dev_state *dev_state,
+		    struct spdk_nvme_health_information_page *page,
+		    const struct spdk_nvme_ctrlr_data *cdata)
+{
+	int	written;
+
+	dev_state->bds_warn_temp_time = page->warning_temp_time;
+	dev_state->bds_crit_temp_time = page->critical_temp_time;
+	dev_state->bds_ctrl_busy_time = page->controller_busy_time[0];
+	dev_state->bds_power_cycles = page->power_cycles[0];
+	dev_state->bds_power_on_hours = page->power_on_hours[0];
+	dev_state->bds_unsafe_shutdowns = page->unsafe_shutdowns[0];
+	dev_state->bds_media_errors = page->media_errors[0];
+	dev_state->bds_error_log_entries = page->num_error_info_log_entries[0];
+	dev_state->bds_temperature = page->temperature;
+	dev_state->bds_temp_warning = page->critical_warning.bits.temperature ?
+		true : false;
+	dev_state->bds_avail_spare_warning = \
```

Comment:
(style)  Avoid unnecessary line continuations

Reply:
I think you could avoid these line continuations by declaring an intermediate variable for the critical_warning info, e.g.

```C
        union   spdk_nvme_critical_warning_state        cw;

...
       cw = page->critical_warning;
       dev_state->bds_temp_warning = cw.bits.temperature ? true : false;
       dev_state->bds_avail_spare_warning = cw.bits.available_spare ?
               true : false;
```